pxtsim/svg.ts

```diff
@@ -152,18 +152,18 @@ namespace pxsim.svg {
         });
     }
 
-    export function mkLinearGradient(id: string): SVGLinearGradientElement {
+    export function mkLinearGradient(id: string, vertical = true): SVGLinearGradientElement {
```

Comment:
default arguments should always be false IMO. so ``vertical`` -> ``horizontal``

Reply:
Sure